concepts/bools/introduction.md

```diff
@@ -1,21 +1,23 @@
 # Introduction
 
-Python represents true and false values with the `bool` type. There are only two values: `True` and `False`. These values can be bound to a variable:
+Python represents true and false values with the `bool` type.
+ There are only two values under that type: `True` and `False`.
+  These values can be bound to a variable:
 
 ```python
-true_variable = True
-false_variable = False
+>>> true_variable = True
+>>> false_variable = False
 ```
 
 We can evaluate boolean expressions using the `and`, `or`, and `not` operators.
```

Comment:
```suggestion
We can evaluate Boolean expressions using the `and`, `or`, and `not` operators.
```

This was not part of the change, and there may be some references to "boolean" vs "Boolean" that I am not seeing…